### PR TITLE
Deployment configuration.

### PR DESCRIPTION
**Pull request by pawfa**
Added configuration for docker image and heroku deployment. Added TEST environment.

### CODE REVIEW

**Comment by pawfa** [comment]
Because of wrong registry links.

File: src/components/types.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { SimulationLinkDatum, SimulationNodeDatum } from 'd3-force';
 
 export enum Environment {
+    TEST = 'TEST',
     SIT = 'SIT',
```
Comment (Felipe444) [inline src/components/types.ts]: you can remove SIT and UAT

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+language: node_js
```
Comment (Felipe444) [inline .travis.yml]: you can add travis and scrutinizer badge

File: docker/nginx.conf [hunk 3]
```diff
@@ -0,0 +1,11 @@
+server {
+  listen ${PORT};
```
Comment (lukaszfiszer) [inline docker/nginx.conf]: I think customization of port that nginx listen to is redundant - it should just use 8080 (or some other port).

When starting docker container you can always bind this port to some other port on the host machine.

**Pull request merged**